.travis.yml

```diff
@@ -1,5 +1,4 @@
 language: node_js
 node_js:
   - "0.12"
-  - "0.11"
-  - "0.10"
+  - "5"
```

Comment:
Interesting! ESLint added support for "6" recently so I was curious: https://github.com/eslint/eslint/commit/36da05264b94ae80d1cde172bae02b1f77808c4b